webapi/src/main/scala/org/knora/webapi/responders/admin/PermissionsResponderADM.scala

```diff
@@ -776,9 +840,12 @@ class PermissionsResponderADM(responderData: ResponderData) extends Responder(re
      * @param requestingUser
      * @return a [[DefaultObjectAccessPermissionGetResponseADM]]
      */
-    private def defaultObjectAccessPermissionGetRequestADM(projectIri: IRI, groupIri: Option[IRI], resourceClassIri: Option[IRI], propertyIri: Option[IRI], requestingUser: UserADM): Future[DefaultObjectAccessPermissionGetResponseADM] = {
-
-        // FIXME: Check user's permission for operation (issue #370)
+    private def defaultObjectAccessPermissionGetRequestADM(projectIri: IRI,
+                                                           groupIri: Option[IRI],
+                                                           resourceClassIri: Option[IRI],
+                                                           propertyIri: Option[IRI],
+                                                           requestingUser: UserADM
+                                                          ): Future[DefaultObjectAccessPermissionGetResponseADM] = {
```

Comment:
Still needs a check if the user is project admin in the project that the permission is about.

Reply:
This validation can be added to the messages in another PR as mentioned above.